packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -908,6 +930,148 @@ class _LargeTitleNavigationBarSliverDelegate
   }
 }
 
+/// The large title of the navigation bar.
+///
+/// Magnifies on over-scroll when [strech] parameter
```

Comment:
there's a typo on `strech`. But also, `[stretch]` is not going to know what to hyperlink to, so it should probably be something like `[CupertinoSliverNavigationBar.stretch]`